pyanaconda/ui/gui/spokes/keyboard.py

```diff
@@ -543,7 +543,7 @@ def on_down_clicked(self, button):
             return
 
         store.swap(cur, nxt)
-        if flags.can_touch_runtime_system("reorder runtime X layouts", touch_live=True):
+        if conf.system.can_localize_live:
             self._flush_layouts_to_X()
```

Comment:
This can be `can_configure_runtime_X_layouts` as well.

Reply:
Changed to `can_configure_keyboard`.